components/Form/index.js

```diff
@@ -12,8 +12,8 @@ function Form({ onAddToDo }) {
     <>
       <StyledForm onSubmit={handleSubmit}>
         <label htmlFor="todo" />
-        <p>To do:</p>
-        <StyledInput type="text" id="todo" name="name" required={true} />
+        To do:
```

Comment:
Please don't put any text without an HTML element:

```suggestion
<label htmlFor="todo">To do:</label>
```

Reply:
I changed it. Is it alright if I merge then?

Reply:
Please refer to the [How to Capstone Description](https://neuefische-students.slack.com/archives/C04JU8U7Y7R/p1679666514852089): As soon as you get a green tick, you post your deployment to the #quality-assurance channel.